interfaces.go

```diff
@@ -67,6 +67,22 @@ type ChainStateReader interface {
 	NonceAt(ctx context.Context, account common.Address, blockNumber *big.Int) (uint64, error)
 }
 
+// SyncProgress gives progress indications when the node is synchronising with
+// the Ethereum network.
+type SyncProgress struct {
+	StartingBlock uint64 // Block number where sync began
+	CurrentBlock  uint64 // Current block number where sync is at
+	HighestBlock  uint64 // Highest block alleged number inthe chain
+	PulledStates  uint64 // Number of state trie entries already downloaded
+	KnownStates   uint64 // Total number os state trie entries known about
+}
+
+// ChainSyncReader wraps access to the node's current sync status. If no sync is
+// in progress, the sync progress is assumed to be nil.
```

Comment:
My issue is that this doc is attached to the interface, not the method. Hence it's weird to say it "returns". The interface doesn't return anything.